packages/docs/src/content/03-components/overlays/modal/examples/size.tsx

```diff
@@ -116,7 +116,9 @@ export default () => {
           </Heading>
           <Content>
             <Section>
-              <Heading>Widget-Sichtbarkeit</Heading>
+              <Heading level={3}>
```

Comment:
hier bitte keine level setzen, ich klär mit UX ob wir das evtl direkt in die Section einbauen, falls dass in jedem Modal so sein soll

Reply:
Haben da im Daily zu gesprochen - das sollte in jeder Section innerhalb eines Modals so sein. So wäre die Hauptüberschrift des Modals eine H2, alle Sections innerhalb sind ja quasi ene "Unterkategorie" und damit dann eine H3.

Reply:
Dann nehme ich das explizite Setzen aus diesem PR raus und erstelle dazu ein Issue